src/button/classes.scss

```diff
@@ -25,6 +25,22 @@
   &[disabled] {
     @include iui-button-disabled;
   }
+
+  &.iui-notification-primary {
```

Comment:
Not sure but maybe instead of offering only 4 types of notifications maybe it can be only 1 iui-notification but the user can pass in the color they want it to be so that way they aren't limited to only 4 colors. We can also offer a list of the default colors in react (primary, positive, warning, negative) but also let them put in whatever color they want.

Reply:
Would it be possible for a user to define a custom color with the way the CSS is setup currently?  They'd need to override `&::before { background-color: t(iui-color-foreground-#{$status}); }` & the `box-shadow` color within the pulse animation.  But I think their override would be tied to a status name (primary, positive, warning, negative), right?

Reply:
But do we want to offer custom colors? I think this can be achieved using CSS vars like in `Badge`.

Reply:
If it could be useful to make notifications of any color, I was thinking the notifications would not be tied to status at all. 
Instead of choosing between iui-notification-primary, iui-notification-positive, iui-notification-warning, or iui-notification-negative, we only offer 1 option iui-notification and have a variable passed in --color (green, orange, red, etc.) that sets the color of the notification

Reply:
@bentleyvk or @saaaaaally could you take a look at the code I pushed just now & suggest how I could use CSS vars like in badge.  The notification dot color does change with the theme, so it might not be as easy as it was in badge because the badge color doesn't change with the theme.

Reply:
If we really really want to support custom colors, then you do like this:
![image](https://user-images.githubusercontent.com/36186912/135208709-b546b452-7f01-4a5a-b416-28384242a3c5.png)
![image](https://user-images.githubusercontent.com/36186912/135208727-cb0c896e-4f1c-4004-9923-f6bcf1103a84.png)

But I think we might need to have separate classes for the colors like `iui-primary` and it would set the variable. I guess something like this:
```scss
.iui-primary {
  @include themed {
    --marker-color: t(iui-color-foreground-primary);
  }
}
```

I haven't really used CSS vars so need to read more and play around.

Reply:
I vote we save custom colors for a future PR upon request.